sparse/core.py

```diff
@@ -1067,7 +1071,7 @@ def __abs__(self):
 
     def exp(self, out=None):
         assert out is None
-        return np.exp(self.maybe_densify())
+        return self.elemwise(np.exp)
```

Comment:
Will this always `err` (given current code)

(not a concern, just asking a question)

Reply:
Yes. But given what we talked about in #10 it's probably best to keep it.